WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -465,7 +465,11 @@ targetted_decisions = {
 			OR = {
```

Comment:
The value changes in this file from vanilla (with the exception of the Indian group change)